src/test/scala/cromwell/engine/WorkflowManagerActorSpec.scala

```diff
@@ -68,7 +68,7 @@ class WorkflowManagerActorSpec extends CromwellTestkitSpec("WorkflowManagerActor
         WorkflowId(UUID.randomUUID()) -> WorkflowRunning)
       val ids = workflows.keys.map(_.toString).toSeq.sorted
       val key = SymbolStoreKey("hello.hello", "addressee", None, input = true)
-      val symbols = Map(key -> new SymbolStoreEntry(key, WdlStringType, Option(WdlString("world"))))
+      val symbols = Map(key -> new SymbolStoreEntry(key, WdlStringType, Option(WdlString("world")), false))
```

Comment:
IMHO boolean parameters should be named.  IntelliJ will warn about this in some cases (probably when there are multiple).